server/data/ideas.js

```diff
@@ -1,111 +1,286 @@
+// ======================================================================
 const { ObjectId } = require('mongodb');
 const { ideas } = require('../config/mongoCollections');
 const { isValidObjectId, isValidStr } = require('../utils');
-const { isValidLookingFor, isValidStatus } = require('../utils/ideas');
 const { isValidProjectName } = require('../utils/projects');
 const { isValidUsername } = require('../utils/users');
 const { isValidTechnologies } = require('../utils/projects');
-const {
-	internalServerErr,
-	badRequestErr,
-	notFoundErr,
-} = require('../utils/index');
+const { internalServerErr, badRequestErr, notFoundErr } = require('../utils/index');
```

Comment:
Are you not using prettier and eslint ??. We are all following eslint and prettier indentation. Make sure you follow the same.